File: app/src/main/java/com/llj/architecturedemo/db/AppDb.java
package com.llj.architecturedemo.db;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import com.llj.architecturedemo.db.dao.MobileDao;
import com.llj.architecturedemo.db.model.MobileEntity;

/**
 * ArchitectureDemo
 * describe:
 * author liulj
 * date 2018/6/6
 */
@Database(
        entities = {MobileEntity.class},
        version = 1
)
public abstract class AppDb extends RoomDatabase {

    abstract public MobileDao mobileDao();
}

File: app/src/p_app/main/java/com/llj/architecturedemo/db/AppDb.kt
package com.llj.architecturedemo.db

import androidx.room.Database
import androidx.room.RoomDatabase

import com.llj.architecturedemo.db.dao.MobileDao
import com.llj.architecturedemo.db.entity.MobileEntity

/**
 * ArchitectureDemo
 * describe:
 * author liulj
 * date 2018/6/6
 */
@Database(entities = [MobileEntity::class], version = 1)
abstract class AppDb : RoomDatabase() {
    abstract fun mobileDao(): MobileDao
}
